src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -1162,8 +1162,11 @@ protected int doRun() throws IOException {
       if (isInTestMode()) {
         modules = modulesSupplierForTesting.get();
       } else {
-        for (JsModuleSpec m : jsModuleSpecs) {
-          checkModuleName(m.getName());
+        if (config.jsonStreamMode == JsonStreamMode.IN
```

Comment:
Per comment from internal reviewer, can this use ```equals()```? Google's internal Java practices guide suggests ```equals()```  for enums unless both sides are nullable